app/views/your-claims/your-claims.html

```diff
@@ -1,110 +1,127 @@
+    <span class="secondary-text">Prison</span>
+    <div class="govuk-!-font-weight-bold" data-cy="prison">{{ displayHelper.getPrisonDisplayName(claims[0].NameOfPrison) }}</div>
   </div>
 
-  {% if canStartNewClaim %}
-  <form action="/your-claims/check-your-information" method="GET">
-      <input id="new-claim" type="submit" class="button heading-large" value="Start a new claim" data-cy="new-claim"/>
-  </form>
-  {% else %}
-  <input id="new-claim" type="button" class="button heading-large" disabled value="Start a new claim" />
-  <br />
-  <p class="text-warning">You can't start a new claim as you have an existing claim in progress</p>
-  <br />
-  {% endif %}
-
-  <br />
-  <a href="/start" class="link">Make a claim for a different prisoner</a>
-
-  <hr>
-
-  <h2 class="heading-medium">
-    Your claim{% if claims[1] %}s{% endif %}
-  </h2>
-
-  <table class="claim-list">
-
-    <tr>
-      <th>Date sent</th>
-      <th>Date of visit</th>
-      <th>Progress</th>
-      <th></th>
-    </tr>
+  <div class="govuk-grid-column-one-third">
+    <span class="secondary-text">Prisoner number</span>
+    <div class="govuk-!-font-weight-bold" data-cy="prisoner-number">{{ claims[0].PrisonNumber }}</div>
+  </div>
 
-    {% for claim in claims %}
-      <tr {% if forEdit(claim.Status) %} class='warning' {% endif %}>
-        <td><span class="date-submmited">Date sent </span>{{ dateHelper(claim.DateSubmitted) }}
-        <td><span class="visit-date">	Date of visit </span>{{ dateHelper(claim.DateOfJourney) }}</td>
-        <td
-          {% if claim.IsAdvanceClaim %}
-            {% if claim.Status == 'APPROVED-ADVANCE-CLOSED' %}
-              >Closed
-            {% elif forEdit(claim.Status, claim.IsAdvanceClaim, claim.DateOfJourney) %}
-              class ="text-warning your-claims"><span class="claim-status">Progress </span>Information needed
-            {% elif claim.Status == 'APPROVED' %}
-              class ="text-pending your-claims"><span class="claim-status">Progress </span>Information needed after visit takes place
-            {% elif claim.Status == 'REJECTED' %}
-              >Rejected
-            {% else %}
-              class ="text-pending your-claims"><span class="claim-status">Progress </span>In progress
-            {% endif %}
+</div>
 
+{% if canStartNewClaim %}
+<form action="/your-claims/check-your-information" method="GET">
+  <br>
+  {{ govukButton({
+    text: "Start a new claim",
+    preventDoubleClick: true,
+    attributes: {
+      "data-cy": "new-claim"
+    }
+  }) }}
+</form>
+{% else %}
+<br />
+  {{ govukButton({
+    text: "Start a new claim",
+    preventDoubleClick: true,
+    disabled: true,
+    attributes: {
+      "data-cy": "new-claim"
+    }
+  }) }}
+<br />
+{{ govukWarningText({
+  text: "You can't start a new claim as you have an existing claim in progress",
+  iconFallbackText: "Warning"
+}) }}
+{% endif %}
+
+<a href="/start">Make a claim for a different prisoner</a>
+
+<br>
```

Comment:
Not causing it to not pass but the front-end dev in me doesn't like using breaks for spacing....should really have a class on whatever is above or below with relevant padding/margin.